### DISCUSSION TITLE
Where are the default JSBSim installation location for linux and macos?

### DISCUSSION THREAD

**Discussion by sryu1** [JSBSim-Team/jsbsim#821] [General]
I'm just working on this project that uses JSBSim and I have to know the default installation location for JSBSim. Thanks :)

**Comment by bcoconni** [comment]
We need to know how you got JSBSim in the first place. The installation location is managed by packages, installers, etc.

**Reply by sryu1** [reply]
https://github.com/JSBSim-Team/jsbsim/releases/tag/v1.1.13
If we get the executable installers from here...

**Reply by bcoconni** [reply]
There are no installers for MacOSX but regarding the Debian packages, they are supposed to install their files at a system level i.e. in `/usr/bin`, `/usr/lib`, etc.

**Comment by seanmcleod** [comment]
I know you're asking specifically about Linux and MacOS, but just to mention that for the Windows version of the installer the default installation directory was changed recently, see commit - https://github.com/JSBSim-Team/jsbsim/commit/0880dbe3597c0cac5c9c1412b826faa63fb0cf90

**Reply by sryu1** [reply]
Does that mean it's not going to install into C:\ anymore and into local appdata?

**Comment by seanmcleod** [comment]
See the comment in the associated issue of the commit - https://github.com/JSBSim-Team/jsbsim/issues/798#issuecomment-1367563189

**Reply by sryu1** [reply]
hmm... ok...

**Comment by seanmcleod** [comment]
Note that this is simply the default that the installer will default to, the user can still select a different directory.
